### PR TITLE
Don't load polyfill fallback

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

Currently, if the request to polyfill.io fails, we load a backup checked-in version of the polyfill.io response. To ensure polyfills load before code is executed, `boot.js` is structured in such a way as to prevent if from importing dependencies until polyfills have been applied. This works, but locks us into Webpack and adds complexity to our JS application.

This change removes the request to the polyfill fallback if polyfill.io fails. If polyfill.io fails the application code will still execute, but on older browsers there will be unhandled errors and the JavaScript will likely stop working. In this situation, the only remedy will be to turn off the polyfill.io switch, at which point we will serve polyfills from our expensive checked-in fallback until the polyfill.io service is restored and the switch is turned back on. 

## What is the value of this and can you measure success?

⬇️  Reduces the complexity in our application bootstrapping.
✨  Restores JavaScript in IE9... was previously not being served at all 😱 

## Does this affect other platforms - Amp, Apps, etc?

No

## Tested in CODE?

- [x] Test in CODE

### CODE REVIEW

**Comment by SiAdcock** [comment]
I mean, the JS fails on IE9 because it doesn't support [`matchMedia`](https://github.com/guardian/frontend/blob/master/static/src/javascripts-legacy/lib/detect.js#L97-L98) and (and it's not included in polyfill.io by default)... But anyway

File: static/src/javascripts/boot.js [hunk 1]
```diff
@@ -1,101 +1,88 @@
...
-                    require => {
-                        raven.context(
-                            { tags: { feature: 'commercial' } },
-                            () => {
-                                markTime('commercial boot');
-                                const commercialBoot = config.switches
-                                    .commercial
-                                    ? require('bootstraps/commercial')
-                                    : Promise.resolve;
+        markTime('commercial request');
```
Comment (regiskuckaertz) [inline static/src/javascripts/boot.js]: I'll be bold and suggest a small refactor:
```
const loadChunk = (n, f) => new Promise(resolve => {
  require.ensure([], () => { resolve(f()); }, n);
});

loadChunk('commercial', () => {
  markTime('commercial boot');
  if (config.switches.commercial) {
    return require('bootstraps/commercial')();
  }
})
.then(loadChunk.bind(null, 'enhanced', () => {
  markTime('enhanced boot');
  if (window.guardian.isEnhanced) {
    require('bootstraps/enhanced/main')();
  }
})
.then(() => {
  if (document.readyState === 'complete') {
    capturePerfTimings();
  } else {
    window.addEventListener('load', capturePerfTimings);
  }
});
```

**Comment by regiskuckaertz** [comment]
Brilliant 🌟

File: static/src/javascripts/boot.js (hunk 1 above)
Comment (sndrs) [inline static/src/javascripts/boot.js]: i don't think webapck is clever enough to understand this. afaik, it handles the webpack-specific `require.ensure` calls by string matching

**Comment by sndrs** [comment]
out of interest, what was the benefit over simply using script tags in the end?

**Comment by SiAdcock** [comment]
@sndrs This method allows you to load scripts asynchronously (non-render-blocking) and also preserve the execution order.

`defer` would do this, but is broken in IE10 and below, and also waits for DOM ready, which could delay execution.

**Comment by sndrs** [comment]
perfect :) cheers, i can never hold all the options in my head at once, really great stuff

**Review by sndrs — APPROVED** [review]

**Pull request merged**